index.js

```diff
@@ -89,6 +90,21 @@ function sitedown (options, callback) {
  */
 function mdToHtml (filePath) {
   var body = fs.readFileSync(filePath, encoding)
+
+  var md = markdownIt(mdOpts)
```

Comment:
The creation of the markdownIt instance had to be moved here in order to ensure link anchors are generated with the same numbering every time.

[github-slugger](https://github.com/Flet/github-slugger) (which markdown-it-github-headings uses) generates links like GitHub does. If there are duplicate links, a number is appended to the end. getting a new instance of markdownIt ensures that numbering is reset on every run. If we didn't do this, the counters would never reset between uses.

Reply:
ok, that makes sense